setup.py

```diff
@@ -33,6 +33,7 @@
     'random2',
     'mock',
     'msgpack-python',
+    'ZopeUndo',
```

Comment:
That import is no longer used after ce623c3b6ede65f7db69aa32dbae55de92582be3

Reply:
removed